satpy/readers/clavrx.py

```diff
@@ -236,13 +272,13 @@ def _read_axi_fixed_grid(filename: str, l1b_attr) -> geometry.AreaDefinition:
         area_extent, ncols, nlines = _CLAVRxHelper._area_extent(x, y, h)
 
         area = geometry.AreaDefinition(
-            "ahi_geos",
-            "AHI L2 file area",
-            "ahi_geos",
-            proj,
-            ncols,
-            nlines,
-            np.asarray(area_extent))
+          f"{sensor}_geos",
```

Comment:
Removes the type np.asarray as requested.  Simplifies the description using f-strings.